cukinia/common_security_tests.d/auditd.conf

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2023, RTE (http://www.rte-france.com)
+# SPDX-License-Identifier: Apache-2.0
+
+cukinia_log "$(_colorize yellow "--- check auditd configuration ---")"
+modules_function=" \
+    insmod \
+    kmod \
+    modprobe \
+    rmmod \
+"
+
+syscall_list=" \
+    ioperm \
+    prctl \
+    ptrace \
+"
+
+file_monitoring="\
+    open \
+    openat \
+    unlink \
+"
+
+as "ANSSI-BP-028-R73 - auditd service is active" \
+    cukinia_systemd_unit auditd
+
+as "ANSSI-BP-028-R73 - auditd is configure to output in syslog" cukinia_cmd \
+    grep -q "active = yes" /etc/audit/plugins.d/syslog.conf
+
+
+for function in ${modules_function}; do
+    /sbin/$function &>/dev/null
+    as "ANSSI-BP-028-R73 - $function call is log" cukinia_cmd \
+        grep -q $function /var/log/audit/audit.log
+done
+
+touch /etc/test
+as "ANSSI-BP-028-R73 - modification in /etc/ is log" cukinia_cmd \
+    grep -q "/etc/test" /var/log/audit/audit.log
+
+mount /dev/loop4 /mnt/ &>/dev/null
+as "ANSSI-BP-028-R73 - mount/umount call is log" cukinia_cmd \
+    grep -q "mount" /var/log/audit/audit.log
+
+for syscall in ${syscall_list}; do
+    python3 /usr/bin/$syscall".py" &>/dev/null
+    as "ANSSI-BP-028-R73 - $syscall call is log" cukinia_cmd \
+        grep -q $syscall /var/log/audit/audit.log
+done
+
+rm /etc/test
+as "ANSSI-BP-028-R73 - file deletion is log" cukinia_cmd \
```

Comment:
logged

Reply:
Is it only in /etc ? specify that
Also use the less generic file name
Move adjacent to the other /etc test

Reply:
All file deletion are logged. The idea was to delete the file `/etc/cukinia-auditd-test` created by the previous test.